azurerm/internal/services/lighthouse/lighthouse_definition_resource_test.go

```diff
@@ -26,6 +26,11 @@ func TestAccLighthouseDefinition_basic(t *testing.T) {
 	// ObjectId for user, usergroup or service principal from second Tenant needs to be set as a environment variable ARM_PRINCIPAL_ID_ALT_TENANT.
 	secondTenantID := os.Getenv("ARM_TENANT_ID_ALT")
 	principalID := os.Getenv("ARM_PRINCIPAL_ID_ALT_TENANT")
+	if secondTenantID == "" || principalID == "" {
+		t.Skip("Skipping as ARM_TENANT_ID_ALT and/or ARM_PRINCIPAL_ID_ALT_TENANT are not specified")
+		return
```

Comment:
Does `t.Skip()` stops the execution already? So no need to add another `return` here?